src/Topics/StandardModernPrecision/TwoDiamondOpeners.hs

```diff
@@ -261,8 +259,7 @@ minimumResponse = let
       in
         situation "min" action B.b2D2N3C explanation
   in
-    wrap $ return sit <~ T.allVulnerabilities
-                      <~ [T.North, T.West]  -- We're both unpassed hands
+    stdWrapSE sit  -- We're both unpassed hands
```

Comment:
Note that this one changed who could be the dealer: the old version was wrong, caused by unthinking copy-and-paste!